deployments/ecmps/release-v2.0-fix/Sprint8/6111/camdecmps/functions/rpt_quartely_em_summary_data.sql

```diff
@@ -0,0 +1,331 @@
+                                        ind.fuel_ind,
+                                        -- Indicates whether fuel specific data was reported for the location in the emission report
+                                        max(case when osd.fuel_cd is null then 0 else 1 end) as Fuel_Exists_Ind
+                                from  lst
+                                        join (select 0 as fuel_ind) ind
+                                        on null is null
+                                        join camdecmps.operating_supp_data osd
+                                        on osd.mon_loc_id = lst.mon_loc_id
+                                        and osd.rpt_period_id = lst.rpt_period_id
+                                        and osd.op_type_cd in ( 'BCO2', 'CO2M', 'HIT', 'NOXM', 'NOXR', 'OPDAYS', 'OPHOURS', 'OPTIME', 'SO2M' )
+                                group
+                                    by  lst.mon_plan_id,
+                                        lst.rpt_period_id,
+                                        lst.mon_loc_id,
+                                        ind.fuel_ind
+                            ) sel
+                            join camdecmpsmd.reporting_period prd
+                            on prd.rpt_period_id = sel.rpt_period_id
+                            left join camdecmps.operating_supp_data osd
+                            on osd.mon_loc_id = sel.mon_loc_id
+                            and osd.rpt_period_id = sel.rpt_period_id
+                            and osd.op_type_cd in ( 'BCO2', 'CO2M', 'HIT', 'NOXM', 'NOXR', 'OPDAYS', 'OPHOURS', 'OPTIME', 'SO2M' )
+                            and (
+                                    -- Gets non fuel specific data for non fuel rows
+                                    sel.fuel_ind = 0 and osd.fuel_cd is null
+                                    or
+                                    -- Gets Op Hours for fuel rows when no fuel specific data exists for the emission report
+                                    sel.fuel_ind = 1 and sel.Fuel_Exists_Ind = 0 and osd.op_type_cd = 'OPHOURS'
+                                    or
+                                    -- Gets fuel specirfic data for fuel rows
+                                    sel.fuel_ind = 1 and osd.fuel_cd is not null
+                                )
+                    group
+                        by  sel.mon_plan_id,
+                            sel.fuel_ind,
+                            sel.rpt_period_id,
+                            sel.mon_loc_id,
+                            prd.calendar_year,
+                            prd.quarter,
+                            osd.fuel_cd
+                    union   all
+                    --
+                    -- Ozone-Season-to-Date Data for the Non-Fuel-Specific Report Tables
+                    --
+                    select  lst.mon_plan_id,
+                            0 as fuel_ind,
```

Comment:
Since FUEL_IND is not needed in this query because it is always 0, and because it is not needed in the sub-queries with which it is unioned.